src/Components/FilterSideBar/Filters.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react'
+import { useFilter } from '../../Context/FilterContext';
+import "../../Pages/Products/Products.css"
+const Filters = () => {
+    const {
+        FilterState: { byCategory, sortBy, byRating },
+        FilterDispatch,
+    } = useFilter();
+    return (
+        <div>
+            <div className="filter-sidebar">
+                <div className="filter-sidebar-heading">
+                    <span>
+                        <b>Filters</b>
+                    </span>
+                    <button
+                        className="ecom-btn"
+                        onClick={() => {
+                            FilterDispatch({ type: "Clear_Filter" });
+                          }}
+                    >
+                        Clear
+                    </button>
+                </div>
+                <div className="filter-cat">
+                    <span>
```

Comment:
In filters, if only one can be selected then use radio button not checkbox. For filter by category, multi select should be there, right now able to select only one. Also star rating should be shown on the card.